components/tenant-mgt/org.wso2.carbon.tenant.mgt.core/src/main/java/org/wso2/carbon/tenant/mgt/core/TenantPersistor.java

```diff
@@ -186,16 +186,7 @@ private int addTenant(Tenant tenant) throws TenantMgtException {
         int tenantId;
         TenantManager tenantManager = TenantMgtCoreServiceComponent.getTenantManager();
         try {
-            if (StringUtils.isBlank(tenant.getAdminFirstName())) {
-                throw new TenantManagementClientException(ERROR_CODE_MISSING_REQUIRED_PARAMETER.getCode(), String
-                        .format(ERROR_CODE_MISSING_REQUIRED_PARAMETER.getMessage(), "owner firstName"));
-            } else if (StringUtils.isBlank(tenant.getAdminLastName())) {
-                throw new TenantManagementClientException(ERROR_CODE_MISSING_REQUIRED_PARAMETER.getCode(), String
-                        .format(ERROR_CODE_MISSING_REQUIRED_PARAMETER.getMessage(), "owner lastName"));
-            } else if (StringUtils.isBlank(tenant.getAdminName())) {
```

Comment:
Do we need to remove tenant.getAdminName() ?

Reply:
Fixed in 64daf4b